kube_apiserver_metrics/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG - Kube_apiserver_metrics
 
+# 3.6.0 / 2023-04-25
+
+* [Added] Add new metric `aggregator_unavailable_apiservice`. See [#14457](https://github.com/DataDog/integrations-core/pull/14457).
+
```

Comment:
Could you revert this change? This file is automatically updated by `ddev` when we release the integrations so you do not need to update it manually 🙂 More info [here](https://datadoghq.dev/integrations-core/guidelines/pr/) if you want to have a look

Reply:
Today I learned something ! Thanks for the review. It should be fixed in the last commit.